asapdiscovery/simulation/scripts/MProSimulation.py

```diff
@@ -0,0 +1,170 @@
+
+
+import sys
+
+# Configure logging
+import logging
+from rich.logging import RichHandler
+FORMAT = "%(message)s"
+from rich.console import Console
+logging.basicConfig(
+    level=logging.INFO, format=FORMAT, datefmt="[%X]", handlers=[RichHandler(markup=True)]
+)
+log = logging.getLogger("rich")
+
+## Preliminary Imports
+import os
+import openmm
+import argparse
+import rdkit
+from rdkit import Chem
+
+## OpenMM Imports
+from openmm.app import PDBFile
+from openmm import unit, app
+from openmm.app import PME, HBonds
+from openmm.app import Modeller
+from openmm import LangevinMiddleIntegrator
+from openmm.app import Simulation
+from openmm import MonteCarloBarostat
+from openmm import Platform
+from openmm.app import DCDReporter
+from openmm.app import Simulation
+import mdtraj
+from mdtraj.reporters import XTCReporter
+from openff.toolkit.topology import Molecule
+from openmmforcefields.generators import SystemGenerator
+
+#Simulation Parameters
+
+steps = 10000
+equilibrationSteps = 100
+reportingInterval = 1250
+dt = 4.0 * unit.femtoseconds
+temperature = 300 * unit.kelvin
+friction = 1.0 / unit.femtoseconds
+pressure = 1.0 * unit.atmospheres
+barostatInterval = 25
+nonbondedMethod = PME
+nonbondedCutoff = 1.0*unit.nanometers
+ewaldErrorTolerance = 0.000001
+constraints = HBonds
+rigidWater = True
+constraintTolerance = 0.000001
+hydrogenMass = 4.0*unit.amu
+platform = Platform.getPlatformByName('CUDA')
```

Comment:
This means that this script will fail to run if the GPU isn't able to use 'CUDA'! I would suggest making sure it fails quickly if so.